### PR TITLE
spess library ruin (abandoned, too bored for this pr)

### PR DESCRIPTION
**Pull request by Seris02**
## About The Pull Request

title

# except with more books

## Why It's Good For The Game

more space ruins

also

a fun place for spess explorers to get lots of wood and probably make a base

## Changelog
:cl:
add: spess library
/:cl:

### CODE REVIEW

**Comment by Arturlang** [comment]
This just looks like a perfect cult base, but it really could be a bit more interesting, maybe with a little story or a bit more of something around it, instead of just a floating room.

File: tools/mapmerge2/requirements.txt [hunk 1]
```diff
@@ -1,3 +1,3 @@
-pygit2==0.27.2
+pygit2==1.1.1
 bidict==0.13.1
-Pillow==6.2.0
+Pillow==7.0.0
```
Comment (Ghommie) [inline tools/mapmerge2/requirements.txt:3]: When you told me you were gonna bump up the mapmerge requirements, I thought you were going to do that in a separate PR (hell even a webedit), you lazy.

Comment (Seris02) [reply tools/mapmerge2/requirements.txt:3]: ...I literally asked if I could include this in my pr and you said yes

**Comment by YakumoChen** [comment]
Theres already a library ship that exists in space, this kind of feels extremely poorly done and isnt really anywhere near sensible to what should be found in space.

**Comment by YakumoChen** [comment]
Its also an uninspired giant square metal tumor that contains entirely too little for how much space it takes up. There's dozens of ruins that make decent bases (the library ship is actually a pretty great spot) so I dont see how this is any improvement.

**Review by Ghommie — CHANGES_REQUESTED** [review]
What the person above said, this is a relic-less square box of rustiness, not really a good thing for deep space ruins.

**Comment by Seris02** [comment]
eh, don't merge yet, imma edit it for more interest and less space
